src/main/java/com/atlauncher/data/Settings.java

```diff
@@ -295,6 +300,21 @@ public void convert(Properties properties) {
             analyticsClientId = importedAnalyticsClientId;
         }
 
+        String importedPreLaunchCommand = properties.getProperty("prelaunchcommand");
```

Comment:
You can remove this section as this is just here for people on older versions of the launcher to convert their properties based configuration to json based. Since this is a new feature, this part isn't necessary.